brunch-config.js

```diff
@@ -33,7 +33,7 @@ var pageConfig = require('./src/app/js/page_config.js');
 exports.config = {
   paths: {
     /* dir to build to */
-    'public': 'www',
+    'public': 'www/demo',
```

Comment:
This is an important change!

It makes the built app+cms available at /www/demo/ after build.. (was /www/)